docker-compose.ci.env

```diff
@@ -0,0 +1,111 @@
+######### LOCAL DEV #########
+REBUILD=1                            # Set to anything to rebuild when running locally in Docker
+FETCH_DEPS=                         # Set to anything to fetch deps when rebuilding locally in Docker
+
+
+######### SHARED #########
+POSTGRES_PORT=5432                                                              # (Required) Should almost always be 5432
+POSTGRES_USER=test                                                              # (Required) The user to use to connect to the db
+POSTGRES_PASSWORD=test                                                          # (Required) The password to use to connect to the db
+
+
+######### L2 GETH VARS #########
+TARGET_GAS_LIMIT
+VOLUME_PATH=/mnt/l2-node/l2
+HOSTNAME=geth_l2
+PORT=8545
+NETWORK_ID=108
+
+
+######### MICROSERVICES #########
+### DEPLOYMENT ###
+SHOULD_DEPLOY_CONTRACTS=1
+L1_CONTRACT_DEPLOYMENT_PRIVATE_KEY=df8b81d840b9cafc8cd68cf94f093726b174b5f109eba11a3f2a559e5f9e8bce
+L1_SEQUENCER_ADDRESS=0xaaa6c3FBA9be28B4e4777Cdcb9Ed781199083358
+L1_CONTRACT_OWNER_ADDRESS=0xbbb3F0b64561dC149bf9c10e95f2D10AE552F25b
+FORCE_INCLUSION_PERIOD_SECONDS=1500
+
+LOG_NEW_LINES=1                                                                 # Logs new lines instead of <\n>
+
+# Container
+STARTUP_WAIT_TIMEOUT=30                                                         # The amount of times to attempt to connect to L2 geth before failing (set to -1 if not to wait for L2 geth).
+
+# Logging
+DEBUG=info*,error*,warn*,debug*                                                 # The comma-separated logging patterns to match (common options are `error*`, `info*`, `warn*`, and `debug*`)
+
+# Postgres
+POSTGRES_HOST=postgres                                                          # (Required) The host DNS entry / IP for the postgres DB
+POSTGRES_DATABASE=rollup                                                        # (Required) The database name to connect to (should be `rollup`)
+POSTGRES_CONNECTION_POOL_SIZE                                                   # The connection pool size for postgres (defaults to 20)
```

Comment:
Just having `POSTGRES_CONNECTION_POOL_SIZE` but not set to anything will be ok?

Reply:
Yep!
```
 # The connection pool size for postgres (defaults to 20)
```
☝️ it defaults to 20 when not set, and declaring the var but not providing a value means it's still not set. 

Just leaving it (and others like it) in there as documentation of what _could be_ set if a user chooses.